etc/vendor-libmongoc.sh

```diff
@@ -103,11 +103,13 @@ echo "RENAMING header files"
                   \
                   -e '/private/! s+include "mongoc.h"+include "CLibMongoC_mongoc.h"+' \
                   -e '/private\|defs/! s+include "mongoc-+include "CLibMongoC_mongoc-+' \
+                  -e '/private/! s+include "mongoc/+include "CLibMongoC_+' \
                   -e '/private/! s+include <mongoc-+include <CLibMongoC_mongoc-+' \
                   -e '/private/ s+include "mongoc/+include "+' \
                   \
                   -e 's+CLibMongoC_utlist.h+utlist.h+' \
-                  -e 's+PRId64+\"lld\"+'
+                  -e 's+PRId64+\"lld\"+' \
+                  -e 's+include <common-thread-private.h>+include "CLibMongoC_common-thread-private.h"+'
```

Comment:
nit: I tried to group the sed rules by folder, `common` goes before `bson` and `mongoc`

Reply:
fixed